dockyard/api/app.py

```diff
@@ -0,0 +1,42 @@
+import pecan
+
+from dockyard.api import config as api_config
+from oslo_config import cfg
```

Comment:
I think some requirement file also need to be created which specify the dependency modules(ex. oslo_config) for running dockyard.